src/main/java/Study_hw_1/CarManager.java

```diff
@@ -0,0 +1,63 @@
+package Study_hw_1;
+
+import java.util.Scanner;
+
+public class CarManager {
+    private Car user[];
+    private String username;
+    private int serveral; // 게임 횟수
+
+    CarManager(String username){
+        this.username=username;
+    }
+
+    public void setServeral(int serveral) {
+        this.serveral = serveral;
+    }
+
+    private int countuser(){
+        int count=1;
+        for(int i=0; i<username.length();i++)
+        if(username.charAt(i)==',') count ++;
+        return count;
+    }
+    // 유저가 몇명인지 카운트 해주는 메소드
+
+    private void setUser(){
+       user=new Car[countuser()];
+        for(int i=0;i<countuser();i++)
+            user[i]=new Car(username.split(",")[i]);
+    }
+
+    public void play(){
+        setUser();
+        System.out.println("실행 결과");
+        for(int i=0;i<serveral;i++){
+            for(int j=0;j<countuser();j++){
```

Comment:
모든 Car 객체를 이동시키고 있네요. 메서드로 분리해보면 어떨까요?
```suggestion
            doRaceOneTime();
```